superset-frontend/src/views/CRUD/data/database/DatabaseModal.tsx

```diff
@@ -69,11 +83,15 @@ const StyledInputContainer = styled.div`
 
   .input-container {
     display: flex;
-    align-items: center;
+    align-items: top;
 
     label {
       display: flex;
       margin-right: ${({ theme }) => theme.gridUnit * 2}px;
```

Comment:
I think you'll want to remove this as well, but double check when you have the other labels moved into the correct label tag.